File: src/pkg/runtime/plan9/386/signal.c
// Copyright 2010 The Go Authors.  All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file. 

#include "runtime.h"

int64
runtime·nanotime(void) 
{
	// Won't compile.
}

String
runtime·signame(int32)
{
	return runtime·emptystring;
}

void
runtime·resetcpuprofiler(int32 hz)
{
	// TODO: Enable profiling interrupts.
	
	m->profilehz = hz;
}

File: src/cmd/go/internal/tool/signal.go
// Copyright 2026 The Go Authors. All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.

//go:build !plan9 && !js

package tool

import (
	"os"
	"syscall"
)

var signalsToForward = []os.Signal{syscall.SIGHUP, os.Interrupt, syscall.SIGQUIT, syscall.SIGTERM}
